cypress/e2e/jwt_login.js

```diff
@@ -0,0 +1,49 @@
+const loginUrl = 'http://localhost:9000/metagenomics/login';
+const homePageUrl = 'http://localhost:9000/metagenomics';
+const myDataPageUrl = 'http://localhost:9000/metagenomics/mydata';
+const username = 'webin-66688';
+const password = 'DEBSzfWkQL64S@4';
```

Comment:
I assume this is not a read username and password... but just in case I'm flagging it